meterpreter/meterpreter/src/main/java/com/metasploit/meterpreter/core/core_transport_set_timeouts.java

```diff
@@ -0,0 +1,54 @@
+package com.metasploit.meterpreter.core;
+
+import com.metasploit.meterpreter.Meterpreter;
+import com.metasploit.meterpreter.TLVPacket;
+import com.metasploit.meterpreter.TLVType;
+import com.metasploit.meterpreter.Transport;
+import com.metasploit.meterpreter.Utils;
+import com.metasploit.meterpreter.command.Command;
+
+public class core_transport_set_timeouts implements Command {
+
+    public int execute(Meterpreter meterpreter, TLVPacket request, TLVPacket response) throws Exception {
+        Transport currentTransport = meterpreter.getTransports().current();
+
+        try {
+            long sessionExpiry = request.getIntValue(TLVType.TLV_TYPE_TRANS_SESSION_EXP);
```

Comment:
Sorry, was on the mobile earlier.

Another option was to add an overload for `getIntValue` which returned a default value of `0`, just like the other Meterpeters do. I just went with what the current implementation does, and that is it does the usual enterprise java thing and throw exceptions unnecessarily :)